buildenv/jenkins/docker-slaves/x86/centOS/Dockerfile

```diff
@@ -0,0 +1,215 @@
+  && sed -i 's/#RSAAuthentication.*/RSAAuthentication yes/' /etc/ssh/sshd_config \
+  && sed -i 's/#PubkeyAuthentication.*/PubkeyAuthentication yes/' /etc/ssh/sshd_config
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+# Download and install boot JDK from AdoptOpenJDKs
+
+RUN ln -s java-1.7.0 /usr/lib/jvm/java-7-openjdk-amd64
+
+RUN mkdir /usr/lib/jvm/adoptojdk-java-80 \
+  && cd /usr/lib/jvm/adoptojdk-java-80 \
+  && wget -O bootjdk8.tar.gz "https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl=openj9&os=linux&arch=x64&release=latest&type=jdk" \
+  && tar -xzf bootjdk8.tar.gz \
+  && rm -f bootjdk8.tar.gz \
+  && mv $(ls | grep -i jdk) bootjdk8 \
+  && mv bootjdk8/* /usr/lib/jvm/adoptojdk-java-80
+
+RUN mkdir /usr/lib/jvm/adoptojdk-java-90 \
+  && cd /usr/lib/jvm/adoptojdk-java-90 \
+  && wget -O bootjdk9.tar.gz "https://api.adoptopenjdk.net/v2/binary/releases/openjdk9?openjdk_impl=openj9&os=linux&arch=x64&release=latest&type=jdk" \
+  && tar -xzf bootjdk9.tar.gz \
+  && rm -f bootjdk9.tar.gz \
+  && mv $(ls | grep -i jdk) bootjdk9 \
+  && mv bootjdk9/* /usr/lib/jvm/adoptojdk-java-90
+  
+RUN mkdir /usr/lib/jvm/adoptojdk-java-10 \
+  && cd /usr/lib/jvm/adoptojdk-java-10 \
+  && wget -O bootjdk10.tar.gz "https://api.adoptopenjdk.net/v2/binary/releases/openjdk10?openjdk_impl=openj9&os=linux&arch=x64&release=latest&type=jdk" \
+  && tar -xzf bootjdk10.tar.gz \
+  && rm -f bootjdk10.tar.gz \
+  && mv $(ls | grep -i jdk) bootjdk10 \
+  && mv bootjdk10/* /usr/lib/jvm/adoptojdk-java-10
+
+RUN cd /home/${USER} \
+  && wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz \
+  && tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2 \
+  && rm -f freemarker.tgz
+
+RUN cd /usr/src \
+  && wget http://curl.haxx.se/download/curl-7.24.0.tar.bz2 \
+  && tar xfj curl-7.24.0.tar.bz2 \
+  && rm -f curl-7.24.0.tar.bz2 \
+  && cd curl-7.24.0 \
+  && ./configure --prefix=/usr \
+  && make \
+  && make install 
+
+RUN mkdir /home/${USER}/openjdk_cache \
+  && cd /home/${USER}/openjdk_cache \
+  && git init --bare \
+  && git remote add jdk8 https://github.com/ibmruntimes/openj9-openjdk-jdk8.git \
+  && git remote add jdk9 https://github.com/ibmruntimes/openj9-openjdk-jdk9.git \
+  && git remote add jdk10 https://github.com/ibmruntimes/openj9-openjdk-jdk10.git \
+  && git remote add jdk11 https://github.com/ibmruntimes/openj9-openjdk-jdk11.git \
+  && git remote add jdk https://github.com/ibmruntimes/openj9-openjdk-jdk.git \
+  && git remote add openj9 https://github.com/eclipse/openj9.git \
+  && git remote add omr https://github.com/eclipse/openj9-omr.git \
+  && git fetch --all
+
+# Expose SSH port and run SSHD
+EXPOSE 22
+RUN mkdir -p /var/run/sshd \
+  && service sshd start \
+  && service sshd stop
+  
+CMD ["/usr/sbin/sshd","-D"]
```

Comment:
Add comment: `# Enable SSH daemon program to provide secure encrypted communications.`